services/conference/src/scripts/components/footer/footer.tsx

```diff
@@ -30,53 +32,135 @@ const useStyles = makeStyles((theme) => {
       transform: 'scale(0.5)',
       margin: '1.2em 0 0 -2.1em',
     },
+    itemSelected: {
+      color: 'primary',
+    },
+    item:{
+    },
   })
 })
 
 
 export const Footer: React.FC<BaseProps> = (props) => {
   const classes = useStyles()

```

Comment:
Component 'Footer' seems be able to split into three child component: MicButton, SpeakerButton, VideoButton. Every button contains two fabs and one menu.